app/models/user_group.rb

```diff
@@ -129,6 +129,10 @@ def self.council_group_wac
     UserGroup.find_by(metadata_id: GroupsMetadataCouncils.find_by(friendly_id: 'wac').id, metadata_type: 'GroupsMetadataCouncils')
   end
 
+  def self.teams_committees_group_wst
+    GroupsMetadataTeamsCommittees.find_by(friendly_id: 'wst').user_group
+  end
```

Comment:
Where is this group (metadata) even coming from? I thought we had previously only migrated the empty groups, not the "TeamMember" roles themselves...

Reply:
We had created the user_groups, but they are connected with converted team roles. So this will work here.